packages/@ourworldindata/grapher/src/lineCharts/LineChart.tsx

```diff
@@ -499,6 +524,9 @@ export class LineChart
                 ? `% change since ${formatColumn.formatTime(startTime)}`
                 : unitLabel
         const subtitleFormat = subtitle === unitLabel ? "unit" : undefined
+        const footer = sortedData.some((series) => series.isProjection)
+            ? `Values are extrapolations projected from available data.`
```

Comment:
I would change the wording a tiny bit to make it more explicit that this applies to striped values only, and not to all values in the tooltip:

Prepend one of: striped/dashed/hatched/shaded